lib/models/patch/multi-file-patch.js

```diff
@@ -44,45 +44,41 @@ export default class MultiFilePatch {
 
   clone(opts = {}) {
     return new this.constructor({
-      buffer: opts.buffer !== undefined ? opts.buffer : this.getBuffer(),
-      layers: opts.layers !== undefined ? opts.layers : {
-        patch: this.getPatchLayer(),
-        hunk: this.getHunkLayer(),
-        unchanged: this.getUnchangedLayer(),
-        addition: this.getAdditionLayer(),
-        deletion: this.getDeletionLayer(),
-        noNewline: this.getNoNewlineLayer(),
-      },
+      patchBuffer: opts.patchBuffer !== undefined ? opts.patchBuffer : this.getLayeredBuffer(),
       filePatches: opts.filePatches !== undefined ? opts.filePatches : this.getFilePatches(),
     });
   }
 
+  getLayeredBuffer() {
```

Comment:
🚨 I should really rename this to `getPatchBuffer()` for consistency. `getLayeredBuffer()` is a relic of the pre-PatchBuffer model.